msal/src/main/java/com/microsoft/identity/client/IPublicClientApplication.java

```diff
@@ -162,4 +171,39 @@ void acquireToken(@NonNull final Activity activity,
         void onError(final MsalException exception);
     }
 
+    /**
+     * Callback object used in Device Code Flow.
+     * This callback provides the following methods for communicating with the protocol.
+     *                 1). Receiving authentication information (user_code, verification_uri, and instruction message)
```

Comment:
The formatting of this javadoc looks like it might be chewed up by the code formatter :/

Can you do a`ctrl` + `alt` + `l` on this and confirm it won't get munged? If it doesn't all good, otherwise you might want to use some `<p>` or `<li>` tags to keep it formatted properly if/when formatter is run